README.md

```diff
@@ -1,37 +1,123 @@
+
+This WP PHP Console [WordPress](http://www.wordpress.org) plugin includes and initialises the PHP Console server library and provides a WP PHP Console Settings page to Administrators.
+
+PHP Console allows you to test any WordPress specific function or class (including those introduced by your active theme and plugins!) from Remote PHP Eval Terminal and inspect results, catch error and warnings with call stack trace straight from the Chrome JavaScript console.
+
+PHP Console also allows you to "`print_r()`" PHP variables from the WordPress server to the JavaScript console and optionally filter some of them from the Eval Terminal based on a tag mechanism.
+
+PHP Console uses an http(s) side channel for communication between Browser and Server.
+The debug information from the server does not appear in the HTML content of your pages but directly in the Browser's JavaScript console.
 
 [![Download from WordPress.org](https://github.com/nekojira/wp-php-console/blob/master/assets/wordpress-download-btn.png)](https://wordpress.org/plugins/wp-php-console/)
 
 
-## Installation
+## == Installation ==
+
+1. First, install [Google Chrome extension PHP Console](https://chrome.google.com/webstore/detail/php-console/nfhmhhlpfleoednkpnnnkolmclajemef) from the [Chrome WebStore](https://chrome.google.com/webstore/search/php%20console?_category=extensions).
+Make sure the PHP Console Chrome extension is enabled through chrome://extensions/.
+
+2. Then, add this plugin to your WordPress installation
+
+  - Install as any other WordPress plugin from your WordPress `Plugins` `Add New` menu or
+
+  - Upload it in `wp-php-console` directory into your `wp-content/plugins/` directory or corresponding plugins directory in your installation
+
+3. Activate the plugin through the `Plugins` menu in WordPress.
+
+4. In the `Settings` menu go to `WP PHP Console`
+
+  - Activate WP PHP Console from WordPress `Plugins` menu as you would do with any other plugin.
+
+  - Then go to `WP PHP Console` settings page from the `Settings` menu. From here you need to enter a password for the Eval Terminal.
+
+  - You can also set other options.
+
+### = Options =
+
+In `Settings` `WP PHP Console` page
+
+- You can tick a checkbox to force PHP Console to a SSL connection (of course then if you don't actually have SSL (https), PHP Console simply won't work).
+
+- You can secure your server by specifying IP addresses to restrict the accessibility from the Eval Terminal
+(a single address eg. `192.168.0.4` or an address mask eg. `192.168.*.*` or multiple IPs, comma separated `192.168.1.22,192.168.1.24,192.168.3.*`).
+In case of having issues connecting with the Remote PHP Eval Terminal, temporarily leave this blank.
```

Comment:
Since the bug fix, using * does not work. In my tests, *.*.*.* , my IP and * work, but * not.